app/bundles/NotificationBundle/Entity/StatRepository.php

```diff
@@ -247,7 +247,8 @@ public function getSentCounts($notificationIds = [], \DateTime $fromDate = null)
         $q->groupBy('s.notification_id');
 
         //get a total number of sent notifications first
-        $results = $q->execute()->fetchAll();
+        $run     = $q->execute();
+        $results = $run->fetchAllAssociative();
```

Comment:
I see this in many places. It's nit-pick but variables should be a noun and not a verb. Can it be returned to the one-liner it was before? If possible in all places this was split.

Reply:
I fixed as it used to be before